src/EnergyPlus/PollutionModule.cc

```diff
@@ -6528,35 +750,21 @@ void GetEnvironmentalImpactFactorInfo(EnergyPlusData &state,
     // SUBROUTINE INFORMATION:
     //       AUTHOR         Linda Lawrie
     //       DATE WRITTEN   August 2008
-    //       MODIFIED       na
-    //       RE-ENGINEERED  na
 
     // PURPOSE OF THIS SUBROUTINE:
     // This routine allows access to data inside this module from other modules (specifically the
     // output tabular reports.
 
-    // METHODOLOGY EMPLOYED:
-    // na
-
-    // REFERENCES:
-    // na
-
-    // USE STATEMENTS:
-    // na
-
-    // Locals
-    // SUBROUTINE ARGUMENT DEFINITIONS:
-    // Each of the arguments must be entered in the EnvironmentalImpactFactors object
-
-    if (state.dataPollutionModule->GetInputFlagPollution) {
+    auto &pm = state.dataPollution;
```

Comment:
[src/EnergyPlus/PollutionModule.cc:758]:(style),[constVariable],Variable 'pm' can be declared as reference to const